cardigan/stories/components/Cards.js

```diff
@@ -41,6 +41,7 @@ const CompactCardExample = () => <CompactCard
   DateInfo={null}
   Tags={null}
   labels={labelList}
+  gaEventV2={null}
```

Comment:
In this particular instance, it's because this is the Cardigan story.

I added the `gaEventV2` as a nullable prop to `CompactCard`s because we don't track anything from existing `CompactCard`s, but they are what the `Outro` is composed of, and we _do_ want to track them when they are part of the `Outro`.

Alternatively the prop itself could be optional, but I think we prefer to make them required and nullable?

Reply:
Feels like a strange implementation.
Is the question we're trying to answer what is the context of the compact card?

I also find it a bit odd that we now start tracking the outro type (read, visit, resesarch), but don't track the content type of the other compact cards, which feel really related.

Is there potential either send the context down to the card (which could be pretty useful elsewhere with multiple lists), or even wrap the card in another tracked component and have an outro event and a compact card event?